src/device/contact.ts

```diff
@@ -191,9 +191,7 @@ export class Contact extends deviceBase {
   async BLEparseStatus(): Promise<void> {
     await this.debugLog('BLEparseStatus')
     // ContactSensorState
-    this.ContactSensor.ContactSensorState = this.serviceData.doorState === 'open'
-      ? this.hap.Characteristic.ContactSensorState.CONTACT_NOT_DETECTED
-      : this.hap.Characteristic.ContactSensorState.CONTACT_DETECTED
+    this.ContactSensor.ContactSensorState = this.getContactSensorState(this.serviceData.doorState)
```

Comment:
I don't use the BLE API, so I'm not sure, but is it okay to use `doorState` in here? 🤔